### PR TITLE
assignment hist tab integration aim TM-2768

### PR DESCRIPTION
**Pull request by SidelineCory24**
[BE PR](https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/759)
[Mock PR](https://github.com/MetaPhase-Consulting/mock-fsbid/pull/343)

To-do:
Show the following under the Assignments History Tab
- [x] Language
- [x] Location
- [x] Position Title
- [x] Position Number
- [x] Skill

### CODE REVIEW

**Comment by SidelineCory24** [comment]
This is currently blocked by a WS issue.

**Comment by SidelineCory24** [comment]
image

File: src/Components/ProfileDashboard/Assignments/AssignmentsListResultsCard/AssignmentsContent/AssignmentsContent.jsx [hunk 1]
```diff
@@ -1,10 +1,11 @@
 import { get, has } from 'lodash';
 // import { Link } from 'react-router-dom';
 import {
-  NO_ASSIGNMENT_STATUS, NO_ASSIGNMENT_TOD_DESC, NO_LANGUAGES,
-  NO_POSITION_NUMBER, NO_POST, NO_SKILL,
+  NO_ASSIGNMENT_DATE, NO_ASSIGNMENT_STATUS, NO_ASSIGNMENT_TOD_DESC,
+  NO_ORG, NO_POSITION_NUMBER, NO_POST, NO_SKILL,
```
Comment (elizabeth-jimenez) [inline src/Components/ProfileDashboard/Assignments/AssignmentsListResultsCard/AssignmentsContent/AssignmentsContent.jsx:5]: let's go ahead and consolidate like we did on the other PR - can just pull in `DEFAULT_TEXT` instead

Comment (SidelineCory24) [reply src/Components/ProfileDashboard/Assignments/AssignmentsListResultsCard/AssignmentsContent/AssignmentsContent.jsx:5]: done

**Comment by elizabeth-jimenez** [comment]
im not seeing languages coming through, but i see it in your SS, looking into it

File: src/Components/ProfileDashboard/Assignments/AssignmentsListResultsCard/AssignmentsContent/AssignmentsContent.jsx [hunk 2]
```diff
@@ -25,34 +23,34 @@ const AssignmentsContent = ({ assignment }) => (
       {/* <Link to={`/archived/${get(assignment, 'position.position_id')}`}>View Position</Link> */}
     </div>
     <div>
-      <span className="bid-list-card-title-post">Location: </span>
-      {getPostName(get(assignment, 'position.post', NO_POST))}
+      <span className="bid-list-card-title-post">Location (Org): </span>
+      {`${getPostName(get(assignment, 'position.post') || DEFAULT_TEXT)} (${get(assignment, 'position.organization') || DEFAULT_TEXT})`}
```
Comment (elizabeth-jimenez) [inline src/Components/ProfileDashboard/Assignments/AssignmentsListResultsCard/AssignmentsContent/AssignmentsContent.jsx:27]: same comment as API - we don't want to default a send to a () call

File: src/Components/ProfileDashboard/Assignments/AssignmentsListResultsCard/AssignmentsContent/AssignmentsContent.jsx [hunk 3]
```diff
@@ -25,34 +23,34 @@ const AssignmentsContent = ({ assignment }) => (
...
     </div>
     <div>
       <span className="bid-list-card-title-post">Skill: </span>
-      {get(assignment, 'position.skill', NO_SKILL)}
+      {get(assignment, 'position.skill') || DEFAULT_TEXT}
```
Comment (elizabeth-jimenez) [inline src/Components/ProfileDashboard/Assignments/AssignmentsListResultsCard/AssignmentsContent/AssignmentsContent.jsx:31]: 🙌

File: src/Components/ProfileDashboard/Assignments/AssignmentsListResultsCard/AssignmentsContent/AssignmentsContent.jsx [hunk 4]
```diff
@@ -25,34 +23,34 @@ const AssignmentsContent = ({ assignment }) => (
...
+        {get(assignment, 'asgd_tod_desc_text') || DEFAULT_TEXT}
       </div>
     }
     <div>
       <span className="bid-list-card-title-post">Start date and End date: </span>
       <StartEnd
-        start={formatDate(assignment.start_date)}
-        end={formatDate(assignment.end_date)}
+        start={formatDate(get(assignment, 'start_date')) || DEFAULT_TEXT}
+        end={formatDate(get(assignment, 'end_date')) || DEFAULT_TEXT}
```
Comment (elizabeth-jimenez) [inline src/Components/ProfileDashboard/Assignments/AssignmentsListResultsCard/AssignmentsContent/AssignmentsContent.jsx:53]: 🦜

File: src/Components/ProfileDashboard/Assignments/AssignmentsListResultsCard/AssignmentsContent/AssignmentsContent.jsx [hunk 5]
```diff
@@ -25,34 +23,34 @@ const AssignmentsContent = ({ assignment }) => (
...
     </div>
     <div>
       <span className="bid-list-card-title-post">Skill: </span>
-      {get(assignment, 'position.skill', NO_SKILL)}
+      {get(assignment, 'position.skill') || DEFAULT_TEXT}
     </div>
     <div>
       <span className="bid-list-card-title-post">Language: </span>
-      {get(assignment, 'position.language', NO_LANGUAGES)}
+      <LanguageList languages={get(assignment, 'position.languages') || []} propToUse="representation" />
```
Comment (elizabeth-jimenez) [inline src/Components/ProfileDashboard/Assignments/AssignmentsListResultsCard/AssignmentsContent/AssignmentsContent.jsx:35]: Nice job transitioning to using LanguageList!

**Review by elizabeth-jimenez — APPROVED** [review]
😸

**Pull request merged**